
object_0:
  [
    {
      eight_point:   
        [
          {id: 0, x: 0.967, y: -0.575, z: 0.82},
          {id: 1, x: 1.431, y: -0.639, z: 0.82},
          {id: 2, x: 1.497, y: -0.184, z: 0.82},
          {id: 3, x: 1.026, y: -0.119, z: 0.82},
          {id: 4, x: 0.967, y: -0.575, z: 0},
          {id: 5, x: 1.431, y: -0.639, z: 0},
          {id: 6, x: 1.497, y: -0.184, z: 0},
          {id: 7, x: 1.026, y: -0.119, z: 0}
        ]
    },
    {
      center_point: [{id: 'c', x: 1.2315, y: -0.3795, z: 0.41}],
      class: 'chair'
    }
  ]

object_1:
  [
    {
      eight_point:
        [
          {id: 0, x: 1.403, y: 0.556, z: 0.845},
          {id: 1, x: 1.496, y: 0.199, z: 0.845},
          {id: 2, x: 1.988, y: 0.347, z: 0.845},
          {id: 3, x: 1.877, y: 0.690, z: 0.845},
          {id: 4, x: 1.403, y: 0.556, z: 0},
          {id: 5, x: 1.496, y: 0.199, z: 0},
          {id: 6, x: 1.988, y: 0.347, z: 0},
          {id: 7, x: 1.877, y: 0.690, z: 0}
        ]
    },
    {
      center_point: [{id: 'c', x: 1.640 , y: 0.392, z: 0.4225}],
      class: 'chair'
    }
  ]


  